CONTRIBUTING.md

```diff
@@ -200,6 +200,13 @@ For datasets in particular, these should be loaded using a `loader` function fro
 
 For methods and metrics, they should be decorated with the appropriate function in `openproblems.tools.decorators` to include metadata required for the evaluation and presentation of results.
 
+For methods and metrics requiring specific packages, one should first check if those packages are included in any of the existing Docker images, and should otherwise add their packages to the requirements.txt file associated with the relevant Docker image. Image-specific methods can then be imported in the method or metric python file (i.e. `f2.py`). Importantly, the import statement must be located in the function that calls the method or metric. If this is the `f2` function, the first few lines of the function may be structured as follows:
```

Comment:
This should be moved to the `### Adding package dependencies` section. Some of it is redundant with what is already there.

Minor correction: "Image-specific ~methods~ packages"